static/app/components/stacktracePreview.tsx

```diff
@@ -289,4 +277,4 @@ const NoStackTraceWrapper = styled('div')`
   min-height: 56px;
 `;
 
-export default withApi(withTheme(StacktracePreview));
+export default withApi(withTheme(StackTracePreview));
```

Comment:
should be able to use withApi and withTheme

Reply:
Oh nice, did not know about those! Will do that!